CRM/Prospect/Helper/CreateProspectConverted.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * CRM_Prospect_Helper_CreateProspectConverted class.
+ */
+class CRM_Prospect_Helper_CreateProspectConverted {
```

Comment:
Will suggest to rename this class to `CRM_Prospect_Helper_ProspectConverted` and the method to `create`
`CRM_Prospect_Helper_ProspectConverted::create` rather than `CRM_Prospect_Helper_ProspectConverted::createProspectConverted`